docs/pages/docs/foundation/inkwell.mdx

```diff
@@ -0,0 +1,81 @@
+import { Tabs } from 'nextra/components';
+import { Widget } from "../../../components/widget.tsx";
+import LinkBadge from "../../../components/link-badge/link-badge.tsx";
+import LinkBadgeGroup from "../../../components/link-badge/link-badge-group.tsx";
+
+# Inkwell
```

Comment:
After some thought, I don't think this should be named `FInkWell`. I think flutter names theirs `FInkWell` because tapping produces a rippling effect. That's what happens when you dip a tip of a pen into an inkwell.